lychee-lib/src/utils/request.rs

```diff
@@ -70,10 +70,19 @@ pub(crate) fn create(
                     credentials,
                 )))
             } else if let InputSource::FsPath(root) = &input_content.source {
-                if is_anchor {
-                    // Silently ignore anchor links for now
-                    Ok(None)
-                } else if let Some(url) = create_uri_from_path(root, &text, base)? {
+                let path = if is_anchor {
+                    // If the link is just an anchor,
+                    // prepend the filename of the file from which the link came.
+                    root.file_name()
+                        .expect("File doesn't have a file name.")
+                        .to_str()
+                        .expect("Filename is invalid unicode")
+                        .to_string()
+                        + &text
```

Comment:
Those `expect`s would cause the program to panic. I think a better way to handle this would be to log a warning (with `warn!`) and just `return Ok(None)` as before?

Reply:
Agreed, it would be nice to avoid the panic.